apps/OboeTester/app/src/main/java/com/google/sample/oboe/manualtest/AudioStreamBase.java

```diff
@@ -81,6 +81,18 @@ String dump(int framesPerBurst) {
 
             return buffer.toString();
         }
+        /**
+         *Converts ints from JNI to human-readable stream state
+         */
+        private String parseState(int stateId) {
+            final String[] STATE_ARRAY = {"Uninit", "Unknown", "Open", "Starting", "Started",
+                    "Pausing", "Paused", "Flushing", "Flushed",
+                    "Stopping", "Stopped", "Closing", "Closed", "Disconnected"};
```

Comment:
I was originally thinking of shorter names. But since you removed the "state = " then these full names should fit. If we have to we can rearrange  some other text to make fit.